protobuf/v1/replica.proto

```diff
@@ -38,6 +38,8 @@ message Replica {
   string poolname = 8; // name of the pool on which replica is present
   ReplicaSpaceUsage usage = 9; // replica space usage
   repeated string allowed_hosts = 10; // host (nqn's) which are allowed to connect to the target
+  bool is_clone = 11; // true: clone replica from snapshot, false: normal replica
+  optional string snapshot_uuid = 12; // snapshot uuid from which the clone replica is created
```

Comment:
Question: Why does this field need to be optional? Is this not a required field to begin with?

Reply:
The same structure will be used in Volume Creation path. In that path, snapshot_uuid will be NULL. This is the reason to make this field optional.

Reply:
Okay. Right, makes sense. Thanks